### PR TITLE
Dashboard redirects

### PR DESCRIPTION
**Pull request by robert-bryson**
Related to https://github.com/GSA/data.gov/issues/4331 and https://github.com/GSA/data.gov/issues/4001. Combines https://github.com/GSA/datagov-website/pull/47 and https://github.com/GSA/datagov-website/pull/48.

This PR will cut dashboard over to catalog `/dcat-us/validator` and `/reports`.

### CODE REVIEW

File: vars.production.yml [hunk 1]
```diff
@@ -4,4 +4,5 @@ memory: 64M
 routes:
   - route: www-redirects-prod-datagov.app.cloud.gov
   - route: climate.data.gov
+  - route: dashboard.data.gov
   - route: www.data.gov
```
Comment (btylerburton) [inline vars.production.yml:8]: > - route: www.data.gov

do we need to remove this line?

Comment (robert-bryson) [reply vars.production.yml:8]: I don't think so. I believe we want `www` traffic routed through this app still, right?

Comment (btylerburton) [reply vars.production.yml:8]: I'm not sure. @GSA/data-gov-support thoughts?

Comment (jbrown-xentity) [reply vars.production.yml:8]: This is really a custom router for us, that we use for any domain/route that needs to be re-routed in the short/medium term. It doesn't matter whether it's catalog, dashboard, climate, etc.

Comment (robert-bryson) [reply vars.production.yml:8]: Sorry, you quoted `www` so I interpreted it as asking if we need the `www` route on the website app. If you mean `dashboard`, I think long term we would want to remove it, yes.

Comment (btylerburton) [reply vars.production.yml:8]: Ha. No, I was uncertain if keeping `www.data.gov` in this app would interfere with catalog. So long as we're using it as James mentioned, I'm clear 👍🏼

**Pull request merged**